src/shared/components/ChannelCover/ChannelCover.tsx

```diff
@@ -0,0 +1,77 @@
+import { BackgroundPattern } from '@/components'
+import { transitions } from '@/shared/theme'
+import React, { useState } from 'react'
+import { CSSTransition, TransitionGroup } from 'react-transition-group'
+import Icon from '../Icon'
+import {
+  CoverImage,
+  EditableControls,
+  EditCoverButton,
+  Media,
+  MediaWrapper,
+  RemoveCoverButton,
+} from './ChannelCover.style'
+
+type BasicChannelCoverProps = {
+  coverPhotoUrl?: string | null
+}
+type EditableProps =
+  | {
+      editable?: false
+      handleEditCover?: never
+      handleRemoveCover?: never
+    }
+  | {
+      editable: true
+      handleEditCover?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+      handleRemoveCover?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+    }
+
+export type ChannelCoverProps = BasicChannelCoverProps & EditableProps
+
+const ChannelCover: React.FC<ChannelCoverProps> = ({ coverPhotoUrl, editable, handleRemoveCover, handleEditCover }) => {
+  const [overlayVisible, setoverlayVisible] = useState(false)
+
+  return (
+    <MediaWrapper>
+      {editable && (
+        <EditableControls
+          withImage={!!coverPhotoUrl}
+          onMouseEnter={() => setoverlayVisible(true)}
+          onMouseLeave={() => setoverlayVisible(false)}
+        >
+          <EditCoverButton onClick={handleEditCover}>
+            <Icon name="camera" />
```

Comment:
Compare what's in Figma:
<img width="1064" alt="Screenshot 2021-02-18 at 12 12 11" src="https://user-images.githubusercontent.com/12646744/108348922-a7669e80-71e2-11eb-8ed7-b36cb337690f.png">

With what we have:
<img width="699" alt="Screenshot 2021-02-18 at 12 12 15" src="https://user-images.githubusercontent.com/12646744/108348941-adf51600-71e2-11eb-8fd4-c9bfc3b9d8de.png">

There's a couple of issues:
1. There are problems with the camera icon itself - it looks worse in our version
2. The border is of different width
3. Our circle is actually an ellipse, it's 40x38